package.json

```diff
@@ -17,22 +17,14 @@
     "clean": "find . -name node_modules -o -name .turbo -o -name .next -o -name dist -type d -prune | xargs rm -rf",
     "codegen:override-prisma": "tsx scripts/addPrismaOverrides.ts",
     "codegen": "run-p codegen:*",
-    "prepublish": "pnpm build",
     "postinstall": "pnpm codegen",
-    "prepack": "tsx scripts/postversion.ts",
```

Comment:
I think that this should already be covered by Changesets - but if you could describe the exact purpose of this script I could verify this further.

Reply:
To make sure all packages are updated in sync by making sure that the packages don't depend on a `^`-prefixed version.

I.e. `@trpc/client` `10.0.0` should depend on `@trpc/server` `10.0.0` (and not `^10.0.0`)